Application Actions/Preview/1.0/default.json

```diff
@@ -0,0 +1,9 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
```

Comment:
Rotating right should progress reading while rotate left should go backwards. This said, we should swap Scroll Down/Up so that:

Rotate Left -> Scroll Up
Rotate Right -> Scroll Down